sky/clouds/azure.py

```diff
@@ -39,9 +39,11 @@
 _DEFAULT_AZURE_UBUNTU_2004_IMAGE_GB = 150
 _DEFAULT_SKYPILOT_IMAGE_GB = 30
 
-_DEFAULT_CPU_IMAGE_ID = 'skypilot:custom-cpu-ubuntu-v2'
-_DEFAULT_GPU_IMAGE_ID = 'skypilot:custom-gpu-ubuntu-v2'
-_DEFAULT_V1_IMAGE_ID = 'skypilot:custom-gpu-ubuntu-v1'
+# TODO(romilb): Switch back to using our custom images after NCCL + Azure issues
+#  are resolved: https://github.com/skypilot-org/skypilot/issues/4448
+_DEFAULT_CPU_IMAGE_ID = 'skypilot:gpu-ubuntu-2204'
+_DEFAULT_GPU_IMAGE_ID = 'skypilot:gpu-ubuntu-2204'
+_DEFAULT_V1_IMAGE_ID = 'skypilot:v1-ubuntu-2004'
```

Comment:
Can we use the old Azure image as the base image for the packer file, so we don't get performance regression for A10 GPU instances?

Reply:
Yes, that's likely the fix for #4448. For A10 this PR is still using `skypilot:custom-gpu-ubuntu-v2-grid` to avoid the perf regression.